### PR TITLE
Button cleanup

### PR DESCRIPTION
**Pull request by kenzieschmoll**
We were using buttons of different styles and heights throughout DevTools. This PR cleans that up.

### CODE REVIEW

File: packages/devtools_app/lib/src/common_widgets.dart [hunk 1]
```diff
@@ -229,12 +227,111 @@ class StopRecordingButton extends StatelessWidget {
...
+  }
+}
+
+class SettingsOutlinedButton extends StatelessWidget {
+  const SettingsOutlinedButton({
+    @required this.onPressed,
+    @required this.tooltip,
+  });
+
+  final void Function() onPressed;
```
Comment (jacob314) [inline packages/devtools_app/lib/src/common_widgets.dart:245]: nit: `void Function()` to VoidCallback?

File: packages/devtools_app/lib/src/common_widgets.dart [hunk 2]
```diff
@@ -229,12 +227,111 @@ class StopRecordingButton extends StatelessWidget {
...
+
+  final ButtonStyle style;
+
+  final double width;
+
+  final String tooltip;
+
+  @override
+  Widget build(BuildContext context) {
+    final button = Container(
```
Comment (jacob314) [inline packages/devtools_app/lib/src/common_widgets.dart:290]: nit: Container --> SizedBox

File: packages/devtools_app/lib/src/dialogs.dart [hunk 3]
```diff
@@ -104,3 +104,21 @@ class DialogApplyButton extends StatelessWidget {
     );
   }
 }
+
+class DialogTextButton extends StatelessWidget {
+  const DialogTextButton({this.onPressed, this.child});
+
+  final void Function() onPressed;
```
Comment (jacob314) [inline packages/devtools_app/lib/src/dialogs.dart:111]: VoidCallback

File: packages/devtools_app/lib/src/dialogs.dart [hunk 4]
```diff
@@ -104,3 +104,21 @@ class DialogApplyButton extends StatelessWidget {
...
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        height: defaultButtonHeight,
+        child: TextButton(
+          onPressed: onPressed,
+          child: child,
+        ));
```
Comment (jacob314) [inline packages/devtools_app/lib/src/dialogs.dart:122]: nit: trailing comma

File: packages/devtools_app/lib/src/landing_screen.dart [hunk 5]
```diff
@@ -154,12 +154,14 @@ class _LandingScreenBodyState extends State<LandingScreenBody> {
...
         ),
-        const Padding(
-          padding: EdgeInsets.only(left: 20.0),
-        ),
-        ElevatedButton(
-          child: const Text('Connect'),
-          onPressed: connectDebounce.invoke,
+        const SizedBox(width: defaultSpacing),
+        Padding(
+          // Padding to center the connect button with the `TextField`.
```
Comment (jacob314) [inline packages/devtools_app/lib/src/landing_screen.dart:159]: if the font size is different will we lose this centering?
Can you use mainAxisAlignment instead to make things look good? or maybe crossAxisAlignment? I'd have to use the layout explorer to remember which.

**Review by jacob314 — APPROVED** [review]
lgtm

File: packages/devtools_app/lib/src/landing_screen.dart (hunk 5 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/landing_screen.dart:159]: rearranged some of these widgets so we could use axis alignment instead.

**Pull request merged**